_std/machinery.dm

```diff
@@ -12,17 +12,19 @@
 #define LIGHT_BURNED 3
 
 //apc stuff
-#define EQUIP 1
-#define LIGHT 2
-#define ENVIRON 3
-#define TOTAL 4	//for total power used only
+#define EQUIP 1 	//! Power Channel: Equipment
+#define LIGHT 2 	//! Power Channel: Lighting
+#define ENVIRON 3 //! Power Channel: Enviroment
+#define TOTAL 4		//! For total power used only
```

Comment:
those won't work, you need to do the `///` style

Reply:
Huh, it worked locally using the released executable. 😕  Must only partially work with that syntax... I'll poke it again during my lunch break.

Reply:
Line numbers are different... otherwise appear identical.
https://github.com/Azrun/goonstation/commit/9964153a00089275604351f005e3ef08fe9890bf

What am I missing?

Reply:
Maybe @SpaceManiac might know more? It never worked for me when I used it in live docs.

Reply:
what works and what doesn't unfortunately differs between `#define` and other declarations such as vars

Reply:
Thanks!